packages/paymasters/test/ProxyDeployingPaymaster.test.ts

```diff
@@ -319,8 +324,9 @@ contract('ProxyDeployingPaymaster', ([senderAddress, relayWorker, burnAddress])
       await paymaster.setRelayHub(testHub.address)
       const tx = await testHub.callPostRC(paymaster.address, context, gasUseWithoutPost, relayRequest.relayData)
       const gasUsedWithPost = gasUseWithoutPost + gasUsedByPost
+      const hubConfig = await testHub.getConfiguration()
       // Repeat on-chain calculation here for sanity
-      const actualEtherCharge = (100 + parseInt(relayRequest.relayData.pctRelayFee)) / 100 * gasUsedWithPost
+      const actualEtherCharge = (100 + parseInt(hubConfig.pctRelayFee.toString())) / 100 * gasUsedWithPost
```

Comment:
this calculation ignore baseRelayFee

Reply:
this test is a bit abandoned, and baseFee is set to 0